onnx/defs/math/defs.cc

```diff
@@ -2006,6 +2007,301 @@ ONNX_OPERATOR_SET_SCHEMA(
           defs::math::utils::MatMulShapeInference(ctx, 0, 1);
         }));
 
+static const char* MatMulNBits_ver23_doc = R"DOC(
+MatMulNBits is a MatMul with weight quantized with N bits (e.g., 2, 3, 4, 5, 6, 7). It does Matrix Multiplication like
+[MatMul](#matmul) with differences:
+
+1. Input `B` is a 2D constant Matrix. Its input feature count and output feature count are specified by attributes 'K'
+    and 'N'.
```

Comment:
nit: this `N` is different from the `N` is first line (which is the number of bits in quantized type). May be change it to M? Also, change "input feature count" to "input feature count (number of rows)" and likewise for other dim.

Reply:
You are correct this N is the from and `MatMul((M,K) (K,N))` of the weight matrix. In the first line it is listed as N bits I could change the first line to N-Bits or NBits to help differentiate.  I think most people working with MatMul for 2D matrix multiplication are quite familiar with M, K, N matrix notation.

Reply:
That's fine, as long as the names are distinct, thanks.